src/util/index.js

```diff
@@ -37,17 +37,18 @@ export const formatDatePair = (startTime, endTime) => {
 
   if (endTime) {
     if (end.year() !== start.year()) {
-      to = end.format(' - ddd, MMM D, YYY [at] h:mmA z')
+      to = end.format('ddd, MMM D, YYYY [at] h:mmA z')
     } else if (end.month() !== start.month()) {
-      to = end.format(' - ddd, MMM D [at] h:mmA z')
+      to = end.format('ddd, MMM D [at] h:mmA z')
     } else if (end.date() !== start.date()) {
-      to = end.format(' - ddd, MMM D [at] h:mmA z')
+      to = end.format('ddd, MMM D [at] h:mmA z')
     } else {
-      to = end.format(' - h:mmA z')
+      to = end.format('h:mmA z')
     }
+    to = returnAsObj ? to : ' - ' + to
   }
 
-  return from + to
+  return returnAsObj ? { from, to } : from + to
```

Comment:
when `returnAsObj` is true, return `from` and `to` in an object instead of a concatenated string